packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -2,10 +2,13 @@ import { GradationMap } from "@opentripplanner/types";
 import React, { ReactElement } from "react";
 import styled from "styled-components";
 import { Wheelchair } from "@styled-icons/foundation/Wheelchair";
+import { useIntl } from "react-intl";
+import { InvisibleAdditionalDetails } from "../styled";
 
 interface WrapperProps {
   border: boolean;
   color: string;
+  leg: boolean;
```

Comment:
This prop appears to be unused and can be removed.

Reply:
Good catch thank you! [cd5f97e](https://github.com/opentripplanner/otp-ui/pull/675/commits/cd5f97eab5b5b479276c3dac8ef3d5721bfb4252)